kms/softkms/softkms.go

```diff
@@ -176,3 +177,10 @@ func (k *SoftKMS) CreateDecrypter(req *apiv1.CreateDecrypterRequest) (crypto.Dec
 		return nil, errors.New("failed to load softKMS: please define decryptionKeyPEM or decryptionKey")
 	}
 }
+
+func filename(s string) string {
+	if u, err := uri.ParseWithScheme(string(apiv1.SoftKMS), s); err == nil {
```

Comment:
Should this take `Scheme`, and put `Scheme = string(apiv1.SoftKMS)` somewhere for consistency?

Reply:
I've added support for all of them in 04cb894